fuel-txpool/Cargo.toml

```
@@ -21,5 +21,8 @@ thiserror = "1.0"
 tokio = { version = "1.21", default-features = false, features = ["sync"] }
 tracing = "0.1"
 
+[dev-dependencies]
+fuel-core-interfaces = { path = "../fuel-core-interfaces", features = ["test-helpers"] }
```

We could instead just enable test-helpers for this crate, that way any modifications to the test-helpers feature in the future would automatically get captured.

```suggestion
fuel-txpool = { path = ".", features = ["test-helpers"] }
```

---

Right, that should be cleaner